### PR TITLE
Update dependency ol to v7

### PR DESCRIPTION
**Pull request by bcgov-wps**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [ol](https://openlayers.org/) ([source](https://togithub.com/openlayers/openlayers)) | dependencies | major | [`^6.5.0` -> `^7.0.0`](https://renovatebot.com/diffs/npm/ol/6.9.0/7.0.0) |

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Renovate will not automatically rebase this PR, because other commits have been found.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] If you want to rebase/retry this PR, click this checkbox. ⚠ **Warning**: custom changes will be lost.

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

# Test Links:
[Percentile Calculator](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2227.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

File: .github/workflows/zap_scan.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   zap_scan_ocp4:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```
Comment (andrea-williams) [inline .github/workflows/zap_scan.yml:9]: Wasn't this ubuntu update in a previous PR?

**Review by andrea-williams — APPROVED** [review]

File: .github/workflows/zap_scan.yml (hunk 1 above)
Comment (Sybrand) [reply .github/workflows/zap_scan.yml:9]: For a different workflow. I'm sneaking it in here.

Comment (andrea-williams) [reply .github/workflows/zap_scan.yml:9]: I will confess now that I approved this without testing it. Shame on me. Will play around with it now.

Comment (andrea-williams) [reply .github/workflows/zap_scan.yml:9]: Tested it out in dev - map seems to be working a-ok.

**Review by Sybrand — COMMENTED** [review]
I'm waiting for it to process some forecasts to validate that morecast is working correctly.

**Review by conbrad — APPROVED** [review]

**Pull request merged**